activerecord/lib/active_record/fixtures.rb

```diff
@@ -921,14 +921,13 @@ def teardown_fixtures
       ActiveRecord::Base.clear_active_connections!
     end
 
-    def enlist_fixture_connections
-      ActiveRecord::Base.connection_handler.connection_pools.values.map(&:connection)
+    def ats_fixture_connections
+      [ActiveRecord::Base.connection]
```

Comment:
This is not sufficient. I just haven't had a chance to get this right and I suspected it would change anyway with your updates to ATS